### DISCUSSION TITLE
GitHub Sponsor support for South Korea is not actually usable

### DISCUSSION THREAD

**Discussion by BasixKOR** [community/community#27547] [Other Feature Feedback, Questions, & Ideas]
It seems like GitHub is using Stripe to send payouts, but Stripe is not supported in South Korea (for both companies and individuals) which blocks Korean maintainers from actually using it.

I'm genuinely confused as GitHub officially launched Sponsors to South Korea, yet they haven't really checked if the service would work for new supported countries. Are there any plans to work with Stripe to support South Korea as a supported region? Or could we expect another payment provider to be supported?

Edit: After this post Stripe has updated the region selection and now it includes South Korea.

**Comment by jfullstackdev** [comment]
I think it's automatically being done through GitHub Sponsors, not individual accounts, like in the Philippines too, GitHub will handle all once successfully completed all the requirements

here @Helloyunho from South Korea is already done 

![image](https://user-images.githubusercontent.com/47092464/181670840-548709af-9987-4d5b-af72-a17da79293c6.png)

https://github.com/orgs/community/discussions/27414

you can ask the user at that post too for further guidance

**Reply by BasixKOR** [reply]
Thanks for the link, I've contacted the user and it seems like Stripe country selection has been changed since I've tried to use it.

**Reply by jfullstackdev** [reply]
yes that's the thing, it's all handled by GitHub

**Comment by sinoru** [comment]
It seems like Stripe Connect starts to support South Korea after GitHub Sponsor opened.

You can try remove Stripe Connect account from GitHub Sponsor, and create a new Stripe Connect account for South Korea bank accounts.

<img width="786" alt="image" src="https://user-images.githubusercontent.com/1138433/181680697-40d9d036-2844-4a18-9148-a398ad5693ef.png">

**Reply by jfullstackdev** [reply]
yes that's the thing, it's all handled by GitHub

**Reply by sinoru** [reply]
> yes that's the thing, it's all handled by GitHub

Yeah, but Stripe Connect didn't shows the South Korea or other countries a moment just after the GitHub Sponsor launches. That's why this or [that](https://github.com/orgs/community/discussions/27414) issues happens.

**Reply by jfullstackdev** [reply]
yes, again that's exactly the thing, it's all handled by GitHub,

for example, when GitHub gives swags, they are all handled by GitHub, even the payment and the way to deliver it even if the country, say, the Philippines, does not have the full support for FedEx,
